docs/getting-started/sensor-full-disk-access.asciidoc

```diff
@@ -3,23 +3,91 @@
+
+- `10.14.6+, 10.15, 11.0`: Grant the <<endpoint-endgame-sensor,esensor>> full disk access.
+
+
+**Elastic Endpoint**
+
+No prompts to approve the <<enable-kernel-extension,kernel extension>>, <<system-extension>>, or the <<endpoint-endgame-sensor,elastic-endgame>>, due to installation happening through the {agent}. After installation, Endpoint policies **will fail** to detect events until you approve and enable kernel or system extension loading and full disk access for each version, as reflected in the **Administration > Endpoints** page of the security application.
+
+- `10.13, 10.14, 10.15`: <<enable-kernel-extension,kernel extension>>.
+
+- `11.0` (Big Sur): <<system-extension>>, as well as `co.elastic.systemextension`.
+
+- - `10.14.6+, 10.15, 11.0`: <<endpoint-endgame-sensor,elastic-endgame>>
+
+[discrete]
+[[system-extension]]
+== System extension
+
+To fully protect endpoints from malware and other cybersecurity threats when using Elastic Endpoint with system extensions, full-disk access must be enabled for the system extension during <<install-endpoint,installation>> on macOS Big Sur (11.0) and later.  
+
+--
+image::images/system-extension-prompt.png[System Extension Prompt]
+--
+
+If you select **Ok** and continue installation, you'll still need to enable Full Disk Access for the system extension.
 
 1. Open the **System Preferences** application.
 +
-2. Click **Security and Privacy**. On the Security and Privacy panel, select the **Privacy** tab.
+2. Click **Security and Privacy**. On the Security and Privacy panel, select the **Privacy** tab. 
 +
 3. In the left pane, select **Full Disk Access**.
 +
 --
 image::images/select-fda.png[Select Full Disk Access]
 --
 +
-4. In the lower-left corner of the panel, click the **Lock button** and enter your username and password. You can now add the `elastic-agent` or `esensor` file.
+4. In the lower-left corner of the panel, click the **Lock button** and enter your username and password. 
++
+5. Click the + button to view Finder. Find the system extension `com.endgame.systemextension` (Endgame Sensor) or `co.elastic.systemextension` and select. 
+
+The system extension now has full disk access. However, for both the {agent} and Elastic Endgame sector to detect events from a macOS host, you must enable full disk access for the file most relevant to your security set up.
+
+[discrete]
+[[endpoint-endgame-sensor]]
+== Elastic Endpoint and Endgame sensor
+
+The `elastic-endpoint` files appear after you've downloaded and installed the {agent} with <<install-endpoint,Endpoint Security Integration>>. Similarly, the `esensor` file for Elastic Endgame appears once you've downloaded the sensor on your host. 
+
+
+1. Open the **System Preferences** application.
++
+2. Click **Security and Privacy**. On the Security and Privacy panel, select the **Privacy** tab.
++
+3. In the left pane, select **Full Disk Access**. Find the system extension `com.endgame.systemextension` 
```

Comment:
```suggestion
3. In the left pane, select **Full Disk Access**. Find the system extension `com.endgame.systemextension`. 
```

Reply:
We actually don't need the system extension step in this section, so I removed it.